frontend/src/store/slices/diagnoses/diagnoses.slice.ts

```diff
@@ -0,0 +1,64 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
```

Comment:
This. logic should be in the profile slice
https://github.com/BinaryStudioAcademy/bsa-winter-2021-healthcare/pull/38/files#diff-7b198fa7d2e5f23557a114619cc3751056da4606a748af7fb878c6f11b2fa208